### DISCUSSION TITLE
lab6合成問題

### DISCUSSION THREAD

**Discussion by weixiang1118** [bol-edu/HLS-SOC-Discussions#178] [General]
在使用vivado合成時會看到這些
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/81802711/c0bbb08a-e9a8-4f46-9dd6-e3462583f6df)
在打開這些ip檔案去看，發現都是這些問題，想請問是為甚麼? 謝謝
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/81802711/cea76268-e3a0-4b4b-a32c-b2885de96b13)

**Comment by nthuyouwei** [comment]
我那時候進去vivado的單一按synthesis可以過，然後後面發現run_vivado中處理器開三顆但我自己分配ubuntu只有兩顆，所以改一下就可以過了。

**Reply by weixiang1118** [reply]
請問在tcl檔中 設置處理器個數的指令是甚麼 謝謝

**Reply by nthuyouwei** [reply]
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/145022311/1733e097-0be6-4765-90ae-c569e60a32ab)

**Reply by weixiang1118** [reply]
謝謝大大! 感激不盡
